Artifacts/MsSql/Data/Security/1100-ResourceClaimActionAuthorizations_generated - RLPhase2.sql

```diff
@@ -0,0 +1,526 @@
+DECLARE @ClaimSetId INT;
```

Comment:
From a quick preliminary review of these scripts (I did not attempt to run them or review the final output), it appears to me that you are creating all new tables from scratch (for the renamed and new tables in the design), and you are also inserting new security metadata from scratch (based on what data exactly?) rather than doing any kind of data migration.

@vimayya What approach is needed here? Given the broader audience and the likelihood of customized security metadata, it seems that a migration script will be required before it can be included in any release. Also want to note that we'll need additional scripts for PostgreSQL support.